java/core/src/main/java/co/worklytics/psoxy/gateway/impl/CommonRequestHandler.java

```diff
@@ -173,16 +174,20 @@ HttpRequestFactory getRequestFactory(HttpEventRequest request) {
         Optional<String> accountToImpersonate =
            request.getHeader(ControlHeader.USER_TO_IMPERSONATE.getHttpHeader());
 
+
         accountToImpersonate.ifPresent(user -> log.info("Impersonating user"));
         //TODO: warn here for Google Workspace connectors, which expect user??
 
         Credentials credentials = sourceAuthStrategy.getCredentials(accountToImpersonate);
-        HttpCredentialsAdapter initializer = new HttpCredentialsAdapter(credentials);
+        HttpCredentialsAdapter addCredentials = new HttpCredentialsAdapter(credentials);
```

Comment:
I think that name fits better
```suggestion
        HttpCredentialsAdapter httpCredentialsAdapter = new HttpCredentialsAdapter(credentials);
```